### ISSUE TITLE
MFFS erases the visual gui

### ISSUE THREAD

**Issue by Link-Jon** [ferriarnus/VivecraftForge#217]
After some testing with the direwolf20 1.20 modpack,
For some strange reason [MFFS](https://www.curseforge.com/minecraft/mc-mods/mffs) causes the gui (any gui) to render out of frame to the bottom left in an HMD.

(EDIT: Forgot to mention, but the gui is still intractable at its normal position. You just cant see it.)

Unfortunately this is all the information i have.

MFFS Vers - 5.1.9
Vivecraft    - 1.1.5
Forge        - 47.2.0
Minecraft  - 1.20.1

**Comment by fayer3** [comment]
hm they probably have some post shader or something that messes with the render targets

**Comment by ferriarnus** [comment]
https://github.com/BuiltBrokenModding/MFFS_Classic/blob/d963396d7d344e61b73df7974926c00bd91c8490/src/main/java/dev/su5ed/mffs/render/RenderPostProcessor.java#L43 and https://github.com/BuiltBrokenModding/MFFS_Classic/blob/d963396d7d344e61b73df7974926c00bd91c8490/src/main/java/dev/su5ed/mffs/render/OffscreenRenderTarget.java#L7 could be the reason

**Comment by fayer3** [comment]
@Link-Jon can you check if disabling `enableProjectorModeGlitch` in the mods settings fixes it?

**Comment by Link-Jon** [comment]
Yup, changing changing that setting to false does fix the GUI in VR!
(unsure if i should close issue as the setting now needs to get changed by vivecraft or mffs. unsure if vivecraft can?)

**Comment by fayer3** [comment]
patially fixed with 1.1.12, gui shouldn't be squeezed into the lower corner anymore, but the forcefield previews don't render with the glitch effect enabled

**Comment by fayer3** [comment]
mostly resolved with 1.2.0, glitch affect still doesn't work though
